### DISCUSSION TITLE
Installation without HACS

### DISCUSSION THREAD

**Discussion by insomniux** [RobertD502/home-assistant-iocare#29] [General]
I would like to install the IoCare app to add my Coway 300S to my homeassistant system. However I cannot use **HACS** and the **Addons** are not available (i have a **manual installation** on **Debian OS**, not installed using Docker).  I have copied the coway directory (https://github.com/RobertD502/home-assistant-iocare/tree/master/custom_components/coway) to my custom_components directory but I get stuck at the moment I need to complete the settings in **configuration.yaml**. Searching the forums and github I have not been able to find what to add to this file. 

Could someone help me to manually configure the Coway IoCare in configuration.yaml?

My setup is:
- Home Assistant 2023.1.5
- Frontend-version: 20230110.0
- Python: 3.9.2
- Server: Olinuxino-lime2
- OS: Debian 11
- Zigbee Coordinator: CC2652P SMLIGHT SLZB-05

**Comment by RobertD502** [comment]
Once you've copied over the `coway` directory to your custom_components folder, you'll need to restart Home Assistant. Once Home Assistant is back up and running, the integration is setup using the Home Assistant UI (setup is not done via the configuration.yaml file). Navigate to Home Assistant settings --> Devices & Services and tap on the add integration button. From there search for Coway.

**Reply by insomniux** [reply]
OK, thanks for the confirmation. But that does not seem to work for me. I've restarted Homeassistant, and even rebooted the server but the Coway IoCare does not appear in the list of integrations. There are no remarks in the log concerning misconfiguration or so.

**Reply by RobertD502** [reply]
> OK, thanks for the confirmation. But that does not seem to work for me. I've restarted Homeassistant, and even rebooted the server but the Coway IoCare does not appear in the list of integrations. There are no remarks in the log concerning misconfiguration or so. 
> 
> 
> 
> 

Where is your custom_components folder located?

Also, just to clarify: [HACS](https://hacs.xyz/docs/setup/download/) is available for Core installations (such as yourself), but you do need to be running at minimum 2023.6.0.

**Reply by insomniux** [reply]
The location of my custom_components is:
home/ha/.homeassistant/config/custom_components

Good to know that HACS works in 2023.6, but currently I, unfortunately cannot upgrade, waiting for the moment that the provider of the board enables Debian 12

**Reply by RobertD502** [reply]
Aside from that, I can't offer much help since I don't run a core install. Perhaps post on the Home Assistant community forums and someone that is also using the core installation method may be able to help out.

**Reply by insomniux** [reply]
OK, will do that. Thanks anyway and thanks for providing the Coway integration!
